parsing_tools.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+import re
+from defaults import TIMEZONE_SHIFT, DEFAULT_FILE_TYPE, FILE_TYPES

```

Comment:
I think we should import * from default. That way if we add more defaults the list won't be as long

Reply:
True, but if we add, for example, a 100,000,000  items sized dict to our defaults that comes into play only in statistics....

Reply:
We've decided we'll leave it like that for now